src/main/java/com/minecolonies/coremod/client/gui/WindowHutDeliveryman.java

```diff
@@ -1,31 +0,0 @@
-package com.minecolonies.coremod.client.gui;
```

Comment:
Does the delivery gui only use the AbstractHutWindow ?
Planning to display the list a request at one point here but nvm